spec/requests/api/v1.0/portfolio_items_rbac_spec.rb

```diff
@@ -20,8 +20,14 @@
   describe "GET /portfolio_items" do
     it 'returns status code 200' do
       allow(Insights::API::Common::RBAC::Access).to receive(:new).with('portfolio_items', 'read').and_return(access_obj)
+      allow(Insights::API::Common::RBAC::Access).to receive(:new).with('portfolios', 'read').and_return(access_obj)
       allow(Insights::API::Common::RBAC::Roles).to receive(:assigned_role?).with(catalog_admin_role).and_return(false)
       allow(access_obj).to receive(:process).and_return(access_obj)
+      create(:access_control_entry,
+             :group_uuid   => '123',
+             :permission   => 'read',
```

Comment:
@eclarizio this has moved so we have to use the new construct

Reply:
Ah yeah, that's true. Will fix.